templates/liquibase-pro.template.yaml

```diff
@@ -52,6 +53,8 @@ Metadata:
         Parameters:
           - IsAutomatedTest
     ParameterLabels:
+      CodePipelineS3BucketName:
+        default: Code Pipeline S3 bucket name for storing artifacts
```

Comment:
Minor nit: Slightly different than the label in the other template and CodePipeline should be one word